Marble Blast Platinum/platinum/client/scripts/optionsGui.cs

```diff
@@ -297,7 +297,7 @@
 $Options::Title   ["Graphics", $i  ] = "Material Quality";
 $Options::Type    ["Graphics", $i  ] = "value";
 $Options::Name    ["Graphics", $i++] = "animateBackground";
-$Options::Title   ["Graphics", $i  ] = "Animate Background";
+$Options::Title   ["Graphics", $i  ] = "Preload Levels";
```

Comment:
Unrelated change, please remove

Reply:
Whoops. Gonna make a new pull for this, although I don't really know how to deal with multiple changes in the same file with git.